applications/luci-app-ddns/htdocs/luci-static/resources/view/ddns/overview.js

```diff
@@ -0,0 +1,993 @@
+					_('DDNS Autostart disabled'),
+					E('div', { 'class' : 'cbi-value-description' }, 
+					_("Currently DDNS updates are not started at boot or on interface events.") + "<br>" +
+					_("This is the default if you run DDNS scripts by yourself (i.e. via cron with force_interval set to '0')"))
+				]),]);
+		});
+			
+		for (var i = 0; i < rows.length; i++) {
+			section_id = rows[i].getAttribute('data-sid');
+			cfg_detail_ip = rows[i].querySelector('[data-name="_cfg_detail_ip"]');
+			cfg_update = rows[i].querySelector('[data-name="_cfg_update"]');
+			cfg_status = rows[i].querySelector('[data-name="_cfg_status"]');
+			reload = rows[i].querySelector('.cbi-section-actions .reload');
+			stop = rows[i].querySelector('.cbi-section-actions .stop');
+			cfg_enabled = uci.get('ddns', section_id, 'enabled');
+
+			reload.disabled = (status['_enabled'] == 0 || cfg_enabled == 0);
+			
+			host = uci.get('ddns', section_id, 'lookup_host') || _('Configuration Error');
+			ip =  _('No Data');
+			last_update = _('Never');
+			next_update = _('Unknown');
+			service_status = '<b>' + _('Not Running') + '</b>';
+			
+			if (service[section_id]) {
+				stop.disabled = (!service[section_id].pid);
+				if (service[section_id].ip) 
+					ip = service[section_id].ip;
+				if (service[section_id].last_update) 
+					last_update = _(service[section_id].last_update);
+				if (service[section_id].next_update) 
+					next_update = _(service[section_id].next_update);
+				if (service[section_id].pid)
+					service_status = '<b>' + _('Running') + '</b> : ' + service[section_id].pid;
+			}
+
+			cfg_detail_ip.innerHTML = host + '<br>' + ip;
+			cfg_update.innerHTML = last_update + '<br>' + next_update;
+			cfg_status.innerHTML = service_status;
+		}
+
+		return;
+	},
+
+	load: function() {
+		return Promise.all([
+			uci.load('ddns'),
+			this.callDDnsGetServicesStatus(),
+			callDDnsGetStatus(),
+			this.callDDnsGetEnv(),
+			L.resolveDefault(fs.read('/etc/ddns/services'), ''),
+			L.resolveDefault(fs.read('/etc/ddns/services_ipv6'), ''),
+		]);
+	},
+
+	render: function(data) {
+		var services = data[0];
+		var resolved = data[1];
+		var status = data[2];
+		var env = data[3];
+		var logdir = uci.get('ddns', 'global', 'ddns_logdir') || "/var/log/ddns";
+		
+		var services4 = []; 
+		var services6 = [];
+
+		data[4].split('\n').forEach(function(item) {
+			if (!item.startsWith("#")) {
+				services4.push(item.split('\t')[0].slice(1,-1));
+			}
+		});
+
+		data[5].split('\n').forEach(function(item) {
+			if (!item.startsWith("#")) {
+				services6.push(item.split('\t')[0].slice(1,-1));
+			}
+		});
+
+		var m, s, o;
+
+		m = new form.Map('ddns', _('Dynamic DNS'),);
+
+		var is = m.section(form.NamedSection, 'global', 'ddns', _('Information'));
+
+		s = is;
+
+		o = s.option(form.DummyValue, '_version', _('Dynamic DNS Version'));
+		o.cfgvalue = function() {
+			return status[this.option] || _('Error in gaining value')
```

Comment:
Thats a funny error message :)

Reply:
Well i think the original luci-app-ddns was full of this check to handle case where the app is installed but ddns isn't... since ddns is a dependency of this app i think i can drop this...